newrelic-agent/src/test/java/com/newrelic/agent/config/BaseConfigTest.java

```diff
@@ -26,9 +28,10 @@ public class BaseConfigTest {
     // SystemProperties collections with our key value, so we need to be able to identify this magic test
     // key so we can set up and clean up using @Before and @After methods.
 
-    private static final String PREFIX = "BaseConfig";
```

Comment:
I had to rewrite this test pretty significantly; I don't think this worked as intended with the `NEW_RELIC_` or `newrelic.config.` prefixes (particularly the regex-replace from `newrelic.config` to `new.relic`).